lib/pbench/test/functional/agent/cli/commands/test_config.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+
+
+def test_pbench_agent_config_activate_help():
+    command = ["pbench-agent-config-activate", "--help"]
+    out, err, exitcode = pytest.helpers.capture(command)
+    assert b"Usage: pbench-agent-config-activate [OPTIONS]" in out
+    assert exitcode == 0
+
+
+def test_pbench_agent_config_ssh_key_help():
+    command = ["pbench-agent-config-ssh-key", "--help"]
+    out, err, exitcode = pytest.helpers.capture(command)
+    assert b"Usage: pbench-agent-config-ssh-key [OPTIONS]" in out
+    assert exitcode == 0
+
+
+def test_pbench_config_activate(tmp_path, opt_pbench, pbench_run):
+    """test-14"""
+    pbench_conf = """
+    [pbench-agent]
+    install-dir = %s
+    pbench_run = %s
+    [results]
+    """ % (
+        str(opt_pbench),
+        str(pbench_run),
+    )
+
+    config = tmp_path / "pbench-agent.cfg"
+    config.write_text(pbench_conf)
+    cfg = opt_pbench / "config" / "pbench-agent.cfg"
+
+    conf = opt_pbench / "config"
+    conf.mkdir(parents=True)
+    command = ["pbench-agent-config-activate", str(config)]
+    err, out, exitcode = pytest.helpers.capture(command)
+    assert cfg.exists() is True
```

Comment:
Hmm... to be truly paranoid, I might be inclined to verify `assert cfg.exists() is False` before invoking the command. (Easier than deleting it since we don't expect it.)

Shouldn't ever matter as long as we start with a clean test environment...